flink-runtime/src/main/java/org/apache/flink/runtime/util/ClassLoaderUtil.java

```diff
@@ -121,11 +120,7 @@ public static boolean validateClassLoadable(ClassNotFoundException cnfe, ClassLo
 			String className = cnfe.getMessage();
 			Class.forName(className, false, cl);
 			return true;
-		}
-		catch (ClassNotFoundException e) {
-			return false;
-		}
-		catch (Exception e) {
+		}catch(ClassNotFoundException | Exception  e) /*multi-catch refactor*/ {
```

Comment:
you can remove the `ClassNotFoundException` clause since it is included in the `Exception` clause anway.